src/formio/components/selectboxes.ts

```diff
@@ -0,0 +1,18 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type SelectboxesInputSchema = InputComponentSchema<string, Validator, TranslatableKeys>;
```

Comment:
here too - the (default) value type is not a string or array of strings, but it's actually a weird mapping:

```ts
const selectBoxesValue = {
  'opt1': true,
  'opt2': false,
  'opt3': false,
};
```

I'm not sure if they can be undefined or null - I don't think so, but it would be best to confirm that in the SDK with the debug data.

And the same remark about the default value type `T | T[]` applies here too - this should be limited to just the type of the field because `multiple: true` is never supported for this component type. So I'd compose this one from the lower level building blocks too. Perhaps you can find inspiration in the `file.ts` component type.